storage/src/mmr/benches/append.rs

```diff
@@ -0,0 +1,37 @@
+use commonware_cryptography::{Digest, Hasher, Sha256};
+use commonware_storage::mmr::mem::Mmr;
+use criterion::{criterion_group, Criterion};
+use rand::{rngs::StdRng, RngCore, SeedableRng};
+
+fn bench_append(c: &mut Criterion) {
+    for n in [10_000, 100_000, 1_000_000, 5_000_000, 10_000_000] {
+        c.bench_function(&format!("{}/n={}", module_path!(), n), |b| {
+            b.iter_batched(
+                || {
+                    let mut elements = Vec::with_capacity(n);
```

Comment:
I think I can actually create these once rather than recreating per batch (as they are the same each time.